### ISSUE TITLE
ESPnet V2 accuracy

### ISSUE THREAD

**Issue by HYOJINPARK** [sacmehta/ESPNet#69]
I am really appreciated your code.
In your code, the ESPnet v2 recalls the weight of ImgNet pre-trained one.
So, I remove the line and train from scratch for CityScape.
However, I failed to get the same accuracy with your papers.
I also used lr=0.007 (not 0.009 due to your paper)
When I run train_segmentation.py, the best IOU is 47% in the log of train_segmention.py.
IS the accuracy in your paper training from the pre-train weight?

**Comment by sacmehta** [comment]
The accuracy metric used in Cityscapes evaluation is different from what we use. You need to test accuracy using Cityscapes scripts. 

Which ESPNetv2 model are you trying?

Yes, the models in the below repository uses ImageNet pretrained models. However, for Cityscapes, we didn’t notice much difference if trained from scratch. You need to train longer if you are training from scratch.

**Comment by sacmehta** [comment]
https://github.com/sacmehta/EdgeNets
